.github/workflows/ci.yml

```diff
@@ -23,3 +23,12 @@ jobs:
       run: make shellcheck
     - name: Check distribution
       run: TEST_IMAGE=${{ matrix.os }} VERSION=${{ matrix.version }} make test
+
+  test-install-rootless:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+    - uses: actions/checkout@v3
+    - name: Rootless install
+      run: FORCE_ROOTLESS_INSTALL=1 ./rootless-install.sh
```

Comment:
This is a separate workflow step, because we need to check it outside of container (due to `lsmod`, `iptables` checks)

Reply:
Is it worth adding a comment for that? (i.e. why we need to do separate)

Reply:
Added a comment